Updated README

aflah02 opened the pull request:
I have updated the README with the following - 
1. Added Links for Conda, Cython and AQR.
2. Added a brief description for what all the Dependencies do so people who are unfamiliar with them can atleast get a basic idea about them
3. Corrected PyPI spelling in one instance

aflah02 committed:
Update README.md

aflah02 committed:
Update README.md

Added Links

aflah02 committed:
Update README.md

aflah02 committed:
Update README.md

aflah02 commented:
Hey
If any more changes are needed in the README just comment them below I'll do them as well.
Thanks :)

jreback commented on README.md:
```diff
@@ -100,15 +100,15 @@ pip install pandas
-- [python-dateutil](https://labix.org/python-dateutil)
-- [pytz](https://pythonhosted.org/pytz)
+- [NumPy - Adds support for large, multi-dimensional arrays, matrices and high-level mathematical functions to operate on these arrays](https://www.numpy.org)
+- [python-dateutil - Provides powerful extensions to the standard datetime module, available in Python 2.3+](https://labix.org/python-dateutil)
```
you can remove the , available from datetutil and pytz

aflah02 replied:
Understood will do that, thanks for the suggestions @jreback

aflah02 committed:
Update README.md

Removed the Python Versions beyond which they are available from the documentation for python-dateutil and pytz as advised

aflah02 committed:
Merge pull request #1 from pandas-dev/master

Adding New Commits

aflah02 committed:
Merge pull request #2 from aflah02/master

Adding New Commits

aflah02 commented:
I can't seem to understand why it failed. I think it is unrelated to my changes

aflah02 commented:
@jreback Just wanted to mention I've made the changes but some tests seem to fail however I think they are independent of my changes

jreback commented:
thanks @aflah02 failures are unrelated to your PR
